readme.md

```diff
@@ -499,6 +499,17 @@ An instance of [`CacheableLookup`](https://github.com/szmarczak/cacheable-lookup
 **Note:** This should stay disabled when making requests to internal hostnames such as `localhost`, `database.local` etc.\
 `CacheableLookup` uses `dns.resolver4(..)` and `dns.resolver6(...)` under the hood and fall backs to `dns.lookup(...)` when the first two fail, which may lead to additional delay.
 
+###### family
+
+Type: `0 | 4 | 6`\
```

Comment:
I don't like `0` having some special meaning.

Could we make this:

```suggestion
Type: `'auto' | 4 | 6`\
```

Reply:
Or maybe this to make the values the same type:

```suggestion
Type: `'auto' | 'ipv4' | 'ipv6'`\
```

Reply:
Why not make this an array? `prefferedIpVersion: ['ipv4', 'ipv6']`? It would require us to pass custom `lookup` option internally.

Reply:
> I don't like `0` having some special meaning.
> 
> Could we make this:

I was thinking the same think, but that's going to break some implementation already using that undocumented feature.
If we rename it to `ipVersion` as you suggested I think
> Type: `'auto' | 'ipv4' | 'ipv6'`
Will be perfectly fine.

Reply:
> Why not make this an array? `prefferedIpVersion: ['ipv4', 'ipv6']`? It would require us to pass custom `lookup` option internally.

What would be the benefit of that? Does it solve a problem with the existing option?

Reply:
> Why not make this an array? `prefferedIpVersion: ['ipv4', 'ipv6']`? It would require us to pass custom `lookup` option internally.

The thing is: "There's really a need to have a preference on the IP version?"
When a user can explicit set the IP version or have "4 or 6" I can't find any usage for the "6 or 4" option.
I don't know if implementing it would be a good thing, as you said, we would have to implement a custom `lookup` function.

Reply:
> Does it solve a problem with the existing option?

Is there a problem at all? No. It's just a nitpick. Some people prefer IPv6 over IPv4.

> The thing is: "There's really a need to have a preference on the IP version?"

That's the point. I don't think so. Then what does `auto` mean? It means: if there are no IPv4 addresses found, it will use IPv6. That needs to be documented.

Reply:
> It means: if there are no IPv4 addresses found, it will use IPv6. That needs to be documented.

I think you may have missed it, but i wrote it in the README.md

>  - `auto`: IPv4 (if present) or IPv6